--- meson.build.orig	2024-10-25 09:01:19.382406486 +0100
+++ meson.build	2024-10-25 09:01:22.892835709 +0100
@@ -133,13 +133,16 @@
 # use a linker script to avoid exporting those symbols (this is in addition to
 # Meson using `-fvisibility=hidden` for C and `-fvisibility-inlines-hidden` for
 # C++ code. See gh-15996 for details.
-_linker_script = meson.project_source_root() / 'scipy/_build_utils/link-version-pyinit.map'
-version_link_args = ['-Wl,--version-script=' + _linker_script]
-# Note that FreeBSD only accepts version scripts when -shared is passed,
-# hence we need to pass that to `cc.links` explicitly (flag is already
-# present for `extension_module` invocations).
-if not cc.links('', name: '-Wl,--version-script', args: ['-shared', version_link_args])
-  version_link_args = []
+version_link_args = []
+if not is_windows
+  _linker_script = meson.project_source_root() / 'scipy/_build_utils/link-version-pyinit.map'
+  version_link_args = ['-Wl,--version-script=' + _linker_script]
+  # Note that FreeBSD only accepts version scripts when -shared is passed,
+  # hence we need to pass that to `cc.links` explicitly (flag is already
+  # present for `extension_module` invocations).
+  if not cc.links('', name: '-Wl,--version-script', args: ['-shared', version_link_args])
+    version_link_args = []
+  endif
 endif
 
 generate_f2pymod = find_program('tools/generate_f2pymod.py')
