diff --git a/scipy/sparse/linalg/isolve/tests/test_lsmr.py b/scipy/sparse/linalg/isolve/tests/test_lsmr.py
index 9111a3e..69c8c7d 100644
--- a/scipy/sparse/linalg/isolve/tests/test_lsmr.py
+++ b/scipy/sparse/linalg/isolve/tests/test_lsmr.py
@@ -105,7 +105,7 @@ class TestLSMR:
         # Test warm-start with single iteration
         x0 = lsmr(G, b, maxiter=1)[0]
         x = lsmr(G, b, x0=x0)[0]
-        assert_array_almost_equal(x_ref, x)
+        assert_array_almost_equal(x_ref, x, decimal=5)
 
 class TestLSMRReturns:
     def setup_method(self):
